src/gui/screen_printing_serial.cpp

```diff
@@ -17,7 +17,9 @@ screen_printing_serial_data_t::screen_printing_serial_data_t()
     : IScreenPrinting(string_view_utf8::MakeCPUFLASH((const uint8_t *)caption))
     , octo_icon(this, Rect16((240 - pt_ico().x) / 2, GuiDefaults::RectScreenBody.Top(), pt_ico().x, pt_ico().y), IDR_PNG_serial_printing)
     , last_tick(0)
-    , connection(connection_state_t::disconnected) {
+    , connection(connection_state_t::connected) {
+    IScreenPrinting::ClrMenuTimeoutClose();
```

Comment:
do you really need to fully qualify the method names?
`IScreenPrinting::something` instead of just `something`?